include/logging/log_core.h

```diff
@@ -895,6 +896,31 @@ do {									       \
 
 #include <syscalls/log_core.h>
 
+/** @brief Function pointer table for log output formats.
+ *
+ */
+extern void (*log_format_table[])(const struct log_output*, struct log_msg2*, uint32_t);
+
+/** @brief Set logging format.
+ *
+ * @param backend_name Name of the Backend. Example: log_backend_uart
+ * @param log_type Log format.
+ *
+ * @retval -ENOTSUP If the backend does not support changing format types.
+ *         -EINVAL  If the input is invalid.
+ *			0 for success.
```

Comment:
If you use `@retval`, you will need to use it for all individual return values.

Reply:
Addressed.